sass/edx/_variables.scss

```diff
@@ -307,11 +314,11 @@ $input-btn-padding-x-lg:    1rem !default;
 $input-btn-line-height-lg:  1.5 !default;
 
 $btn-font-weight:                $font-weight-normal !default;
-$btn-box-shadow:                 inset 0 1px 0 rgba($white,.15), 0 1px 1px rgba($black,.075) !default;
+$btn-box-shadow:                 inset 0 1px 0 rgba(theme-color("inverse"),.15), 0 1px 1px rgba(theme-color("darkest"),.075) !default;
 $btn-focus-box-shadow:           0 0 0 3px rgba(theme-color("primary"), .25) !default;
-$btn-active-box-shadow:          inset 0 3px 5px rgba($black,.125) !default;
+$btn-active-box-shadow:          inset 0 3px 5px rgba(theme-color("darkest"),.125) !default;
 
-$btn-link-disabled-color:        $gray-600 !default;
+$btn-link-disabled-color:        theme-color("secondary") !default;
```

Comment:
I guess I don't understand what the secondary color means. I wouldn't have pictured it as being the color for disabled elements.

Reply:
I see that Bootstrap does use secondary for this:

https://getbootstrap.com/docs/4.0/utilities/colors/

I would have expected our secondary color to be our pink.

Reply:
Changed it back as well as the commented out ones.